src/types/character/character-type.ts

```diff
@@ -0,0 +1,17 @@
+import { Stat } from './stat'
+import { Item } from '../shop/item'
+import { Effect } from '../skills/effects'
+
+export type CharacterType = {
+  ownerId: string,
+  name: string,
+  gender: string,
+  level: number,
+  experience: number,
+  credits: number,
+  stats: Stat[],
+  inventorySpace: number,
+  inventoryMax: number,
+  inventory: Item[],
+  ailments: Effect[]
```

Comment:
ailments: what a strange word :o
why not effects?

Reply:
Will do!

Reply:
https://www.wordreference.com/enfr/ailment
Ailment is the English name for what happens to your character during the game that is not their common state. Like poisonned, paralized, stunned, etc.

It's just an effect, but it's applied against a character... whereas effects on an item are applied on the user, not the item itself.

Is my explaination convincing? Or did I get too far? xD

Reply:
@asmodee-bellanie this is your code.
I did not knew the word but anyway you have the final word.
You do what you want and wording is clearly your choice.

Reply:
I renamed it anyway. For the beauty of consistency! XD